pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -1759,6 +1759,15 @@ func parseConfigItems(config *zconfig.EdgeDevConfig, ctx *getconfigContext) {
 			}
 			newGlobalConfig.VdiskGCTime = uint32(i64)
 
+		case "timer.gc.rktGracePeriod":
```

Comment:
cameCaseIsForProgrammersAndNotForUsers

Also, the config variables should be document in docs/CONFIG-PROPERTIES.md

Reply:
Cool. Changed it to timer.gc.rkt.graceperiod. 

Added it to docs/CONFIG-PROPERTIES.md

| timer.gc.rkt.graceperiod | integer in seconds | 300 | grace period to be used in rkt gc command |